nerdlets/pathpoint-nerdlet/components/Header/Header.js

```diff
@@ -56,55 +56,57 @@ const Header = ({
         </div>
         {RenderLogo(logoSetup)}
       </div>
-      <div className="kpi">
-        <RangeDateSelector
-          timeRangeKpi={timeRangeKpi}
-          additionalAction={changeTimeRangeKpi}
-          options={[
-            {
-              label: 'DAY',
-              value: '24 HOURS AGO'
-            },
-            {
-              label: 'WEEK',
-              value: '7 DAYS AGO'
-            },
-            {
-              label: 'MONTH',
-              value: '30 DAYS AGO'
-            },
-            {
-              label: 'YDT',
-              value: '365 DAYS AGO'
-            }
-          ]}
-        />
-        <>
-          {filterKpis.map((kpi, index) => {
-            return (
-              <div
-                key={index}
-                style={{ cursor: kpi.link !== '' ? 'pointer' : 'default' }}
-                onClick={() => {
-                  kpi.link !== '' && window.open(kpi.link);
-                }}
-                className="kpicontent"
-              >
-                <div className="kpicontent--colorgrey kpicontent--size10">
-                  {kpi.shortName}
-                </div>
-                <div className="kpicontent--colorblack kpicontent--size12">
-                  {PrintKPI(kpi)}
+      {filterKpis.length > 0 && (
```

Comment:
@rsamanez What is a valid JSON look like now.

Is this valid?

``` 
{
    "pathpointVersion": "1.5.21",
    "kpis": [],
     "stages": [
         //stage data      
      ]
```


What about a scenario where the kpis is missing completely?

```
{
    "pathpointVersion": "1.5.21",
    // NO KPIS SECTION
     "stages": [
         //stage data      
      ]
```

Reply:
You have to keep an empty array to be valid.
{
    "pathpointVersion": "1.5.21",
    "kpis": [],
     "stages": [
         //stage data      
      ]

The second smple IS NOT A VALID SCHEMA.

Reply:
Okay.  We'll want to make a note in the KPIs documentation.  I will add a small change to the branch which updates the KPI docs.